src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -0,0 +1,106 @@
+(ns status-im.ui2.screens.chat.group-details.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.buttons.button :as quo2.button]
+            [status-im.ui2.screens.chat.group-details.style :as style]
+            [quo2.core :as quo2]
+            [quo2.components.icon :as icons]
+            [utils.re-frame :as rf]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ui.components.list.views :as list]
+            [status-im.ui2.screens.chat.components.contact-item.view :as contact-item]))
+
+(defn back-button []
+  [quo2.button/button {:type                :grey
+                       :size                32
+                       :width               32
+                       :style               {:margin-left 20}
+                       :accessibility-label "back-button"
+                       :on-press            #(do
+                                               (rf/dispatch [:navigate-back]))}
+   [icons/icon :i/arrow-left {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
+
+(defn options-button []
+  [quo2.button/button {:type                :grey
+                       :size                32
+                       :width               32
+                       :style               {:margin-right 20}
+                       :accessibility-label "options-button"}
+   [icons/icon :i/options {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
+
+(defn count-container [count]
+  [rn/view {:style (style/count-container)}
+   [quo2/text {:size   :label
+               :weight :medium
+               :style  {:text-align :center}} count]])
+
+(defn prepare-members [contacts admins]
+  (let [data (atom {:owner   {:title :Owner
+                              :data  []}
+                    :online  {:title :Online
+                              :data  []}
+                    :offline {:title :Offline
+                              :data  []}})]
+    (doseq [i (range (count contacts))]
+      (let [contact (rf/sub [:contacts/contact-by-address (nth contacts i)])
```

Comment:
`(nth contacts 1) -> (second contacts)`

Reply:
Fixed by the above snippet, but what did you mean here? This is an `i` not `1`

Reply:
typo :)